bfs.go

```diff
@@ -66,8 +66,8 @@ type MetaInfo struct {
 	Name        string            // base name of the object
 	Size        int64             // length of the content in bytes
 	ModTime     time.Time         // modification time
-	ContentType string            // optional content type
```

Comment:
Well, it is still optional, as not all the implementations support it. But it should at least be consistent within the implementation